reference/snapshot.md

```diff
@@ -17,8 +17,10 @@ Use cases for volume snapshots include:
 Volume snapshots allow the creation of read-only incremental copies of volumes, enabling you to maintain a history of your data. These volume snapshots possess the following characteristics:
 - **Consistency**: The data stored within a snapshot remains consistent across all replicas of the volume, whether local or remote.
 - **Immutability**: Once a snapshot is successfully created, the data contained within it cannot be modified.
+- **Thin Provisioning**: Thin provisioning will be enabled explicitly when creating the volume. It allows volumes to be allocated with minimal initial capacity and dynamically grow as data is written.
```

Comment:
We need to mention that the when a snapshot is created on a thick-provisioned volume, it gets converted into a thin-provisioned volume.